test/hotspot/jtreg/serviceability/dcmd/vm/SystemDumpMapTest.java

```diff
@@ -36,6 +36,7 @@
  * @summary Test of diagnostic command System.map
  * @library /test/lib
  * @requires (os.family == "linux" | os.family == "windows" | os.family == "mac")
+ * @requires !vm.asan
```

Comment:
If this is easy to fix, e.g. like Jiangli did for the static build, I would rather have a fix inside the test than to completely disable it. Otherwise, a short comment would be good explaining why the test cannot run with asan.

Preexisting: I have no clue why this is excluded for riscv quemu, but SystemMapTest is not, both tests are functionally almost identical.